src/Voron/Impl/Paging/Windows32BitsMemoryMapPager.cs

```diff
@@ -269,6 +269,16 @@ public override I4KbBatchWrites BatchWriter()
             return new Windows32Bit4KbBatchWrites(this);
         }
 
+        public override void DiscardPages(long pageNumber, int numberOfPages)
```

Comment:
However, _because_ we unmap, we may want to discard before unmapping on temp data.
Not sure if we can always do that, though. We rely on map/unmap/map show the same data, so we might need to map, discard and then map.